protocol/x/clob/keeper/orders_test.go

```diff
@@ -515,12 +516,10 @@ func TestPlaceShortTermOrder(t *testing.T) {
 			clobs: []types.ClobPair{
 				constants.ClobPair_Btc,
 			},
-			existingOrders:           []types.Order{},
-			feeParams:                constants.PerpetualFeeParamsNoFee,
-			order:                    constants.Order_Carl_Num0_Id0_Clob0_Buy1BTC_Price5subticks_GTB10,
-			expectedOrderStatus:      types.Undercollateralized,
-			expectedFilledSize:       0,
-			expectedMultiStoreWrites: []string{},
+			existingOrders: []types.Order{},
+			feeParams:      constants.PerpetualFeeParamsNoFee,
+			order:          constants.Order_Carl_Num0_Id0_Clob0_Buy1BTC_Price5subticks_GTB10,
+			expectedErr:    types.ErrOrderWouldExceedMaxOpenOrdersEquityTierLimit,
```

Comment:
this and the test below now fails because of equity tier limit and not collat check